### DISCUSSION TITLE
README

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#21] [备发布站的评论系统]
# README

oldwinterの数字花园    主发布站 |    备发布站 |    源代码仓📩订阅 我的newsletter通讯

https://notes.oldwinter.top/readme

**Comment by eze-is** [comment]
请问放弃logseq的考量是什么呢？目前发现logseq的块引用、大纲日记较为有优势；但在长文写作中弱于obsidian。所以现在我是有些纠结到底主要用哪个

**Reply by oldwinter** [reply]
我也纠结了很久。你说的非常精准，logseq的块引用和大纲，日记几个功能明显优于obsidian。所以我之前才考虑联合用。但发现联合用有几个不兼容的地方也带来了麻烦，尤其是logseq自动生成bak文件，还有改动文件后更新延迟等问题。所以我想还是舍弃其中一个。

选ob主要因为其成熟一些，社区人更多。并且我觉得md存储数据已经成了logseq的性能瓶颈，它后面肯定会改成专有格式存储数据。届时，这就更不是我需要的原生md数据存储了，也就没法儿和这些传统博客以及各平台编辑器直接完美兼容了。

**Reply by eze-is** [reply]
嗯嗯，我也明显感知到logseq性能不足，使用起来不够顺滑；且ob的md能够兼容更多平台，现在也已经全面切换至ob。只要忽略较弱的块引用能力，logseq的大部分优势，在ob上也是可以有一定的替代方案。

**Comment by Joe-rq** [comment]
我就是想夸下，这个库太棒了

**Reply by oldwinter** [reply]
感谢夸奖，能对各位有帮助就好，能看到我这个库是一种缘分~~

**Comment by xiaobaicode6** [comment]
如何联系到您呢?使用您的库中的spaced reptition插件 无法生成卡片!

**Reply by fcjz** [reply]
作者在花园中留下邮箱了吧，应该通过邮箱就可以联系到作者了。
